cmd/neofs-adm/internal/modules/morph/initialize_nns.go

```diff
@@ -88,7 +88,7 @@ func (c *initializeContext) updateNNSGroup(nnsHash util.Uint160, pub *keys.Publi
 	if err != nil || sysFee == 0 {
 		return err
 	}
-	return c.sendCommitteeTx(bw.Bytes(), sysFee, true)
+	return c.sendCommitteeTx(bw.Bytes(), true)
```

Comment:
I wouldn't work this way, the system fee is too big.

Reply:
will continue that work in a separate commit